README.md

```diff
@@ -50,7 +50,7 @@ are supported. Alternative shells include:
 
 On **macOS** and **Linux,** the standard terminal window is supported. Optionally, `bash` command
 auto completion may be enabled by copying the
-[balena-completion.bash](https://github.com/balena-io/balena-cli/blob/master/balena-completion.bash)
+[balena_comp](https://github.com/balena-io/balena-cli/blob/master/completion/bash/balena_comp)
```

Comment:
After changes are made to some source code files and to some sections of the README file, including this section, `npm run build` should be run in order to update the `doc/cli.markdown` file, which is the source of the [reference website](https://www.balena.io/docs/reference/balena-cli/). This PR's CI build is currently failing because the automatic changes to `doc/cli.markdown` (made when `npm run build` is run) were not committed:
```
-[balena-completion.bash](https://github.com/balena-io/balena-cli/blob/master/balena-completion.bash)
+[balena_comp](https://github.com/balena-io/balena-cli/blob/master/completion/bash/balena_comp)
 
** Uncommitted changes found (^^^ diff above ^^^) - FAIL **
Error: "catch-uncommitted": Error: npx failed with exit code 1:
"/Users/balena/.nvm/versions/node/v12.20.0/bin/npx" [catch-uncommitted,--catch-no-git,--skip-node-versionbot-changes,--ignore-space-at-eol]
```
So basically, to fix this:
```
npm run build
git add doc/cli.mardown
git commit
```

Reply:
ok done !